dev/PushNotifications/LRPBackgroundTaskInstance.h

```diff
@@ -1,34 +1,36 @@
-// Copyright (c) Microsoft Corporation and Contributors.
+﻿// Copyright (c) Microsoft Corporation and Contributors.
 // Licensed under the MIT License.
 
 #include <winrt/Windows.ApplicationModel.background.h>
+#include "LRPRawNotification.h"
+#include "PushNotificationDummyDeferral.h"
 
 // Mocks IBackgroundTaskInstance to send raw payloads to packaged apps in
 // PushNotificationBackgroundTask::Run by com activation from PushNotificationsLongRunningProcess
-struct PushBackgroundTaskInstance : winrt::implements<PushBackgroundTaskInstance, winrt::Windows::ApplicationModel::Background::IBackgroundTaskInstance>
+struct LRPBackgroundTaskInstance : winrt::implements<LRPBackgroundTaskInstance, winrt::Windows::ApplicationModel::Background::IBackgroundTaskInstance>
```

Comment:
I still feel that adding acronyms in names should be discouraged. Nobody really outside of our team with specific domain knowledge understands LRP. Maybe we call it LongRunningProcessSourcedTaskInstance which is a clear indication of what this really is?

Reply:
Yup. In general, never use acronyms because people who inherit code later cannot make head or tail of them. AuthManager for example is a very bad name. Auth could be Authentication. It could also be Authorization. 

Its actually better to be wordy than concise and indicate the meaning of the component being built.

Reply:
And when you add the name, also add a comment that this component should only be instantiated from the Long Running Process.